assembl/static2/js/app/pages/landingPageAdmin.jsx

```diff
@@ -198,9 +198,11 @@ class LandingPageAdmin extends React.Component<Props, State> {
   render() {
     const { editLocale, header, section, timelineModuleId } = this.props;
     const saveDisabled = !this.dataHaveChanged();
+    // TODO: Remove this crap after migrating all of landing page to react-final-form
```

Comment:
forgot this?

Reply:
Nope. The code below should be killed off when the rest of the landing page is converted to React Final Form